README.md

```diff
@@ -139,7 +139,7 @@ I would like to thank [Microverse](https://github.com/microverseinc) for this pr
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT) licensed.
```

Comment:
- [ ] Great job working on the requested changes. Unfortunately, when I click on your MIT license link, I am still redirected to an error 404 page. Kindly review this.
![Screen Shot 2023-09-07 at 7 15 58 AM](https://github.com/henask12/OOP-School-Library/assets/62849872/9748322d-7ac1-417c-9ad6-af215e7adef2)